Modules/Module_Processing/Module_Structural/xASL_wrp_GetVolumetrics.m

```diff
@@ -23,53 +23,65 @@ function xASL_wrp_GetVolumetrics(x)
 
 %% ------------------------------------------------------------------------------------------------
 %% 1) File management & store tissue volumes
-catVolFile = fullfile(x.D.TissueVolumeDir, ['cat_' x.P.STRUCT '_' x.P.SubjectID '.mat']); % CAT12 results
-MatFile = fullfile(x.dir.SUBJECTDIR, [x.P.STRUCT '_seg8.mat']); % SPM12 results
-SaveFile = fullfile(x.D.TissueVolumeDir, ['TissueVolume_' x.P.SubjectID '.csv']); % ExploreASL results
-xASL_adm_CreateDir(x.D.TissueVolumeDir);
-
-if xASL_exist(catVolFile, 'file') % for CAT12 segmentation
-    catVol = load(catVolFile);
-    if isfield(catVol.S,'subjectmeasures')
-        GMvol = catVol.S.subjectmeasures.vol_abs_CGW(2)/1000; % volume was in cm^3, /1000 = dm^3 or Liter
-        WMvol = catVol.S.subjectmeasures.vol_abs_CGW(3)/1000;
-        CSFvol = catVol.S.subjectmeasures.vol_abs_CGW(1)/1000;
+fileCAT12Volume = ['cat_' x.P.STRUCT '_' x.P.SubjectID '.mat']; % CAT12 results
+pathCAT12Volume = fullfile(x.D.TissueVolumeDir, fileCAT12Volume);
+
+fileSPM12Volume = [x.P.STRUCT '_seg8.mat']; % SPM12 results
+pathSPM12Volume = fullfile(x.dir.SUBJECTDIR, fileSPM12Volume);
+
+SaveFile = fullfile(x.D.TissueVolumeDir, ['TissueVolume_' x.P.SubjectID '.tsv']); % ExploreASL results
```

Comment:
to avoid doubt - should we rename SaveFile to SaveFileTSV?

Reply:
Done in comment "review suggestions Jan"